### PR TITLE
Jan24 Meeting Action Items

### PR DESCRIPTION
**Pull request by saulakravitz**
BodySite:
  - Qualifier and Laterality extensions
  - Added laterality extensions on Breast example -- MARTIN, please check carefully
  - Dropped radiotherapySessions extension from PlannedCourse and examples
  - Did not drop TreatmentIntent from Phase and Plan, because it wasn't there.

### CODE REVIEW

File: input/fsh/DEF_Rulesets.fsh [hunk 1]
```diff
@@ -40,6 +40,14 @@ RuleSet: Identifiers
   * system 1..1 MS
   * value 1..1 MS
 
+RuleSet: BodySiteQualifierAndLaterality
+* bodySite.extension contains
+     BodyLocationQualifier named locationQualifier 0..*   and
+     LateralityQualifier named lateralityQualifier 0..1
+* bodySite.extension[locationQualifier] ^short = "General location qualifier (excluding laterality) for this bodySite"
+* bodySite.extension[locationQualifier] ^definition = "General location qualifier (excluding laterality) for this bodySite"
+* bodySite.extension[lateralityQualifier] ^short = "Laterality qualifier (excluding laterality) for this bodySite"
```
Comment (mvonsieb) [inline input/fsh/DEF_Rulesets.fsh]: Copy paste errors. Should not say (excluding laterality) on the laterality extension.

File: input/fsh/DEF_Rulesets.fsh [hunk 2]
```diff
@@ -40,6 +40,14 @@ RuleSet: Identifiers
...
+* bodySite.extension[lateralityQualifier] ^definition = "Laterality qualifier (excluding laterality) for this bodySite"
```
Comment (mvonsieb) [inline input/fsh/DEF_Rulesets.fsh]: Same as previous line.

**Review by mvonsieb — CHANGES_REQUESTED** [review]
See comment. Plus the examples should get a laterality in the body site, e.g. on the breast examples. I could help with that but have to find time first.

File: input/fsh/DEF_Rulesets.fsh (hunk 2 above)
Comment (saulakravitz) [reply input/fsh/DEF_Rulesets.fsh]: Ha.... cut/paste was good....source had this error (mCode)....fixed

**Review by mvonsieb — CHANGES_REQUESTED** [review]
Some findings for which I'll provide a fix in a pull request to this branch.

**Review by mvonsieb — APPROVED** [review]

**Pull request merged**